templates/components/password-reset-expired.jsx

```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var IconText = require('./icontext.jsx');
+
+var PasswordResetExpired = React.createClass({
+  render: function() {
+    return (
+      <IconText
+        className="messageBox"
+        iconClass="messageIcon fa fa-exclamation-triangle"
+        header="Seems that the link in your email expired."
```

Comment:
Generally I like casual language, but this is likely a frustrating and confusing moment for users. Let's be more clear and concise:

"The link in your email expired."